pkg/cmd/run/watch/watch.go

```diff
@@ -240,7 +240,7 @@ func renderRun(out io.Writer, opts WatchOptions, client *api.Client, repo ghrepo
 	}
 
 	if annotationErr != nil {
-		return nil, fmt.Errorf("failed to get annotations: %w", annotationErr)
+		fmt.Fprintln(out, fmt.Sprintf("%s failed to get annotations: %s", cs.WarningIcon(), annotationErr))
```

Comment:
Aside from linting error here, I think all the changes make sense 👍

Reply:
oh 😅 I didn't have my linting set up locally, thanks for the heads up. fixed!